### PR TITLE
Refresh old EKS blog post

### PR DESCRIPTION
**Pull request by lblackstone**
TODO

- [x] Update content for .NET and Go

Reviving reverted content from #4718

### CODE REVIEW

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 1]
```diff
@@ -2,143 +2,324 @@
...
 ---
 
-
 Provisioning, managing, and monitoring a Kubernetes cluster is
-no easy task. AWS now offers EKS to ease a lot of that burden -- but
-it's still difficult to get up and running. Pulumi's infrastructure as
-code to the rescue! We can now provision an entire EKS cluster with a
-CLI single gesture, thanks to [the `@pulumi/eks`
-package](https://github.com/pulumi/pulumi-eks). Let's see how.
+no easy task. AWS now offers EKS to ease a lot of that burden -- but
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:17]:
```suggestion
not easy. AWS now offers EKS to reduce that burden -- but
```

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 2]
```diff
@@ -2,143 +2,324 @@
...
+it's still difficult to get up and running. Pulumi's infrastructure as
+code can help! We can now provision an entire EKS cluster with a
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:19]:
```suggestion
code SDKs can help! We can provision an entire EKS cluster with a
```

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 3]
```diff
@@ -2,143 +2,324 @@
...
 
 To get started, download the free and open source
-[Pulumi SDK]({{< relref "/docs/get-started/install" >}}). The SDK includes
-the CLI we'll be using below, and requires AWS credentials to access
-your AWS account and provision resources. If you already have the AWS
-CLI installed and configured, you're already all set to go. Otherwise,
-[check out the docs here]({{< relref "/docs/intro/cloud-providers/aws/setup" >}})
-to set things up.
+[Pulumi SDK]({{< relref "/docs/get-started/install" >}}), which includes
+the `pulumi` CLI we'll be using below. If you don't already
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:30]:
```suggestion
the `pulumi` CLI we'll use below. If you don't
```

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 4]
```diff
@@ -2,143 +2,324 @@
...
 provision your cluster:
 
     $ pulumi new https://github.com/pulumi/apps/eks
 
-Using this template will walk you through configuring a few settings
-such as the region you'd like to deploy into, the
-[Amazon Elastic Compute Cloud](http://aws.amazon.com/ec2) (Amazon EC2)
-instance type to deploy into the cluster, the cluster size, and whether to
-install the Kubernetes Dashboard into your cluster automatically.
+Using this template will walk you through configuring a few settings
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:47]:
```suggestion
This template will walk you through configuring a few settings
```

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 5]
```diff
@@ -2,143 +2,324 @@
...
+such as the region you'd like to deploy into, the instance type, and
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:48]:
```suggestion
such as the region to deploy, the instance type, and
```

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 6]
```diff
@@ -2,143 +2,324 @@
...
+
+{{< /chooser >}}
+
 ## Adding resources, and deploying Helm charts
 
-You can extend or modify this program to change the details of their
+You can extend or modify this program to change the details of the
 Amazon EKS cluster, or to install additional AWS or Kubernetes resources
 to associate with the cluster. For example, if applications running in
 the cluster need access to an Amazon S3 bucket, simply create a new
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:201]:
```suggestion
the cluster need access to an Amazon S3 bucket, create a new
```

File: content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md [hunk 7]
```diff
@@ -2,143 +2,324 @@
...
+
+```go
+bucket, err := s3.NewBucket(ctx, "bucket", &s3.BucketArgs{})
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
 Or, to install some Kubernetes applications automatically into the
```
Comment (spara) [inline content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md:240]:
```suggestion
Or, to install Kubernetes applications automatically into the
```

**Review by spara — CHANGES_REQUESTED** [review]
Some minor changes. This is pretty cool, do you think it would work with EKS Anywhere?

**Comment by lblackstone** [comment]
> This is pretty cool, do you think it would work with EKS Anywhere?

Seems like it should, in principle, but I'm only vaguely familiar with that product line.

**Comment by spara** [comment]
> > This is pretty cool, do you think it would work with EKS Anywhere?
> 
> Seems like it should, in principle, but I'm only vaguely familiar with that product line.

I took another look at it again, and it seems like there's a lot of new information. I think we should republish it next week as an update to the original post dated for next Thursday,  What do you think?

**Comment by lblackstone** [comment]
> I think we should republish it next week as an update to the original post dated for next Thursday, What do you think?

Yes, that sounds good. I'll make a few more updates early next week and then it should be good to go.

**Comment by spara** [comment]
> > This is pretty cool, do you think it would work with EKS Anywhere?
> 
> Seems like it should, in principle, but I'm only vaguely familiar with that product line.



> > I think we should republish it next week as an update to the original post dated for next Thursday, What do you think?
> 
> Yes, that sounds good. I'll make a few more updates early next week and then it should be good to go.

Great! 👍

**Review by spara — APPROVED** [review]

**Comment by lblackstone** [comment]
@spara I updated the publish date, so I think this should be ready to merge on Thursday.

**Pull request merged**